### ISSUE TITLE
Critical Success/Fail issues

### ISSUE THREAD

**Issue by WesleySnipes1** [death-save/maestro#78]
at first it looks very easy and promising, but then it doesn't work. i set the playlists and chose to "play random track" of it but whenever sm1 attacks and fails/success nothing happens.

i'm playing the PF2e module, does the module not support this? hype tracks etc are working

**Comment by JamesDeVore** [comment]
This is due to the if statements on line 274, 279.
`if ((d.options.critical && (d.total >= d.options.critical)) && (criticalSuccessPlaylist && criticalSuccessSound)) {
        Playback.playTrack(criticalSuccessSound, criticalSuccessPlaylist);
    } else if ((d.options.fumble && (d.total <= d.options.fumble)) && (criticalFailurePlaylist && criticalFailureSound)) {
        Playback.playTrack(criticalFailureSound, criticalFailurePlaylist)
    }`

I learned that PF2e does not have values set for d.options.critical / d.options.fumble, on any roll I observed.
A fix would either be to set those values in the pf2e system, or add a check in this module basically saying
`if system == 'pf2e
d.options.critical = 20, d.options.fumble = 1
`
Does not solve the +/- 10 issue, but I have solved that on a fork of mine [here](https://github.com/JamesDeVore/maestro)
which I borrow logic from the module "pf2e quick rolls" to help determine if it is a critical success or not. I need to talk with the maintainer before I try and merge it into this one, since it's really not my work, I just kind of hobbled them together.
